### PR TITLE
update json vietnam language

### PR DESCRIPTION
**Pull request by hophat**
update vietnam language

<sub>
By opening a pull request, I certify that I hold the intellectual property of the code I am submitting,
and I am granting the initial authors of WBO a perpetual, worldwide, non-exclusive, royalty-free, and irrevocable license to this code.
</sub>

### CODE REVIEW

**Comment by lovasoa** [comment]
Hello and thank you for your contribution! Are size, text, grid, menu and zoom all vietnamese words?

File: server/translations.json [hunk 1]
```diff
@@ -375,5 +375,43 @@
...
         "board_name_placeholder": "白板名称",
         "view_source": "GitHub上的源代码"
+    },
+ "vn": {
+        "hand": "Tay",
+        "loading": "Đang tải",
+        "tagline": "Một công cụ vẽ cộng tác trực tuyến miễn phí và mã nguồn mở. Cùng nhau phác thảo những ý tưởng mới trên WBO!",
+        "configuration": "Cấu hình",
+        "collaborative_whiteboard": "Bảng trắng cộng tác",
+        "size": "Size",
```
Comment (lovasoa) [inline server/translations.json:385]: Is this Vietnamese ?

File: server/translations.json [hunk 2]
```diff
@@ -375,5 +375,43 @@
...
+        "zoom": "Zoom",
```
Comment (lovasoa) [inline server/translations.json:386]: Is this Vietnamese ?

File: server/translations.json [hunk 3]
```diff
@@ -375,5 +375,43 @@
...
+        "tools": "Công cụ",
+        "rectangle": "Chữ nhật",
+        "square": "Vuông",
+        "circle": "Tròn",
+        "ellipse": "Hình elip",
+        "click_to_toggle": "Bật/tắt",
+        "menu": "Menu",
```
Comment (lovasoa) [inline server/translations.json:393]: Is this Vietnamese ?

File: server/translations.json [hunk 4]
```diff
@@ -375,5 +375,43 @@
...
+        "text": "Text",
```
Comment (lovasoa) [inline server/translations.json:394]: Is this Vietnamese ?

File: server/translations.json [hunk 5]
```diff
@@ -375,5 +375,43 @@
...
+        "mover": "Di chuyển",
+        "straight_line": "Đường thẳng",
+        "pencil": "Gạch ngang",
+        "grid": "Grid",
```
Comment (lovasoa) [inline server/translations.json:398]: Is this Vietnamese ?

File: server/translations.json [hunk 6]
```diff
@@ -375,5 +375,43 @@
...
+        "click_to_zoom": "Nhấp để phóng to \ Nhấn shift và nhấp để thu nhỏ",
```
Comment (lovasoa) [inline server/translations.json:399]: There should be a `\n` instead of just a `\ `  here

**Review by lovasoa — CHANGES_REQUESTED** [review]

File: server/translations.json (hunk 6 above)
Comment (lovasoa) [inline server/translations.json:399]:
```suggestion
        "click_to_zoom": "Nhấp để phóng to \n Nhấn shift và nhấp để thu nhỏ",
```

File: server/translations.json (hunk 4 above)
Comment (lovasoa) [reply server/translations.json:394]: Google translate suggests "bản văn", is. that correct ?

File: server/translations.json (hunk 5 above)
Comment (lovasoa) [reply server/translations.json:398]: lưới ?

File: server/translations.json (hunk 1 above)
Comment (hophat) [reply server/translations.json:385]: yes. It's vietnamese

File: server/translations.json (hunk 2 above)
Comment (hophat) [reply server/translations.json:386]: yes

File: server/translations.json (hunk 5 above)
Comment (hophat) [reply server/translations.json:398]: In Vietnam, I am used to using common languages ​​in English.

File: server/translations.json (hunk 6 above)
Comment (hophat) [reply server/translations.json:399]: yes

File: server/translations.json (hunk 5 above)
Comment (lovasoa) [reply server/translations.json:398]: Is "grid" a common word in vietnamese ? If it isn't, then we should translate it.

File: server/translations.json (hunk 6 above)
Comment (lovasoa) [reply server/translations.json:399]: So, can you make the change ?

**Comment by lovasoa** [comment]
When you are not sure whether a term should be translated, you can take gimp for a reference: https://l10n.gnome.org/POT/gimp.gimp-2-10/gimp.gimp-2-10.vi.po

I'd like to avoid using english terms in the middle of the vietnamese interface

File: server/translations.json (hunk 1 above)
Comment (lovasoa) [reply server/translations.json:385]: Gimp seems to be using "cỡ". Would that be a better translation ?

The word `size` does not seem to be in the vietnamese dictionary...

https://dictionary.cambridge.org/dictionary/english-vietnamese/size

File: server/translations.json (hunk 3 above)
Comment (lovasoa) [reply server/translations.json:393]: It doesn't seem to be in the dictionary...

**Comment by lovasoa** [comment]
@hophat : Do you have any feedback? Do you think these strings would be more understandable in english to an non-english speaking vietnamese public?

**Comment by hophat** [comment]
> Hello and thank you for your contribution! Are size, text, grid, menu and zoom all vietnamese words?

Yes, they are very popular in Vietnam

**Comment by lovasoa** [comment]
@hophat Ok, I guess you only have to fix the json syntax, then.

File: server/translations.json (hunk 4 above)
Comment (hophat) [reply server/translations.json:394]: "văn bản" doesn't match this thought very well

File: server/translations.json (hunk 1 above)
Comment (hophat) [reply server/translations.json:385]: It is widely used in Vietnam.
Everyone understands it

Comment (hophat) [reply server/translations.json:385]: no one used the word "cở"

File: server/translations.json (hunk 5 above)
Comment (hophat) [reply server/translations.json:398]: It is a very popular word. If put from the "lưới", it will be confusing.

**Comment by lovasoa** [comment]
OK, we are ready to merge, then. Thanks !

**Pull request merged**